src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,29 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+public class CEO {
+
+    private String name;
+
+    private ArrayList<Manager> Managers = new ArrayList<>();
```

Comment:
Konwencja nazewnictwa zmiennych to camelCase. Czyli zaczynamy zmienną z małej litery a każdy następny wyraz z wielkiej. Czyli powinno być `manager`.